tests/system/aiplatform/test_experiments.py

```diff
@@ -276,13 +275,9 @@ def trainer(
         def pipeline(learning_rate: float, dropout_rate: float):
             trainer(learning_rate=learning_rate, dropout_rate=dropout_rate)
 
-        compiler.Compiler().compile(
-            pipeline_func=pipeline, package_path="pipeline.json"
-        )
-
-        job = aiplatform.PipelineJob(
+        job = aiplatform.PipelineJob.from_pipeline_func(
```

Comment:
Preference to leave this test as-is since it's testing a different `PipelineJob` path than the test_pipeline_job.py test added in this PR.

Reply:
Reverted